website/blog/2021-05-20-GAAD-One-Year-Later.md

```diff
@@ -0,0 +1,119 @@
+
+## Documentation Additions
+
+The React Native documentation must be updated to reflect each addition or change to the available APIs. The [new addition to the React Native documentation](https://reactnative.dev/docs/next/accessibilityinfo#getrecommendedtimeoutmillis-android) covered the addition of `getRecommendedTimeoutMillis()` to AccessibilityInfo.
+
+## Community Involvement
+
+We want to thank all the contributors mentioned below who have submitted and merged pull requests as well as those who have reviewed and commented on issues.
+
+### Merged Pull Request
+
+- [Added talkback support for button accessibility: disabled prop #31001](https://github.com/facebook/react-native/pull/31001) - closed by [@huzaifaaak](https://twitter.com/huzaifaaak)
+
+- [feat: set disabled accessibilityState when TouchableHighlight is disabled #31135](https://github.com/facebook/react-native/pull/31135) closed by [@natural_clar](https://twitter.com/natural_clar)
+
+- [[Android] Selected State does not annonce when TextInput Component selected #31144](https://github.com/facebook/react-native/pull/31144) closed by [fabriziobertoglio1987](https://github.com/fabriziobertoglio1987)
+
+- [Added talkback support for TouchableNativeFeedback accessibility: disabled prop #31224](https://github.com/facebook/react-native/pull/31224) closed by [@kyamashiro73](https://twitter.com/kyamashiro73)
+
+- [Accessibility/button test #31189](https://github.com/facebook/react-native/pull/31189) closed by [@huzaifaaak](https://twitter.com/huzaifaaak)
+
+- [feat: add getRecommendedTimeoutMillis section on accessibilityInfo #2581](https://github.com/facebook/react-native-website/pull/2581) closed by [grgr-dkrk](https://twitter.com/dkrk0901)
+
+- [Disable accessibilityState when TouchableWithoutFeedback is disabled #31297](https://github.com/facebook/react-native/pull/31297) by [@crloscuesta](https://twitter.com/crloscuesta)
+
+- [Accessibility Fix Image does not announce "disabled" #31252](https://github.com/facebook/react-native/pull/31252) closed by [fabriziobertoglio1987](https://github.com/fabriziobertoglio1987)
+
+- [Disable TouchableOpacity when accessibility disabled is set #31108](https://github.com/facebook/react-native/pull/31108) closed by [@chakrihacker](https://github.com/chakrihacker)
+
+Thank you to the community members who gave their time in other ways!
+
+Simek, saurabhkacholiya, meehawk, intergalacticspacehighway, chrisglein, jychiao and Waltari10
+
+## Get Involved!
+
+We've come a long way but we're not done yet. We need your support to reach the finish line. Facebook's React Native team has committed to supporting contributors working on gap analysis issues. They will continue to respond to comments on Accessibility issues and triage pull requests. The React Native team is also tackling some of the toughest gap analysis issues. This work includes the correct translation of accessibilityRoles to other languages and specifying error text for specific components.
```

Comment:
> Facebook's React Native team has committed to supporting contributors working on gap analysis issues. 

I wonder if this is a good place to link to the Accessibility Project board. I see there's also mention of it below but I think it wouldn't hurt to link twice. This feels like an opportune time because it concretizes what gap analysis issues are (and where to find them).

Reply:
I think there are a lot links on this page and with it being in the next paragraph I don't think it has to be linked in both paragraphs.